<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.10.0" xml:lang="en-US">
  <compounddef id="namespaceutils_1_1exceptions" kind="namespace" language="Python">
    <compoundname>utils::exceptions</compoundname>
    <innerclass refid="classutils_1_1exceptions_1_1InvalidGamePropertyValue" prot="public">utils::exceptions::InvalidGamePropertyValue</innerclass>
    <innerclass refid="classutils_1_1exceptions_1_1InvalidMapData" prot="public">utils::exceptions::InvalidMapData</innerclass>
    <innerclass refid="classutils_1_1exceptions_1_1InvalidTextureAtlas" prot="public">utils::exceptions::InvalidTextureAtlas</innerclass>
    <innerclass refid="classutils_1_1exceptions_1_1OutOfMapBounds" prot="public">utils::exceptions::OutOfMapBounds</innerclass>
    <innerclass refid="classutils_1_1exceptions_1_1PlayerWithSameNameError" prot="public">utils::exceptions::PlayerWithSameNameError</innerclass>
    <innerclass refid="classutils_1_1exceptions_1_1ZeroOrLessSpawnPlayerAttempts" prot="public">utils::exceptions::ZeroOrLessSpawnPlayerAttempts</innerclass>
    <briefdescription>
    </briefdescription>
    <detaileddescription>
    </detaileddescription>
    <location file="/home/samuel/Documents/Python/Tile-Game/game/utils/exceptions.py" line="1" column="1"/>
  </compounddef>
</doxygen>
